### DISCUSSION TITLE
Create Repository,

### DISCUSSION THREAD

**Discussion by PixelatedPurple** [community/community#146184] [Mobile]
On GitHub Android, the ability to create new repositorys is either non existent or playing hide and seek, on the dashboard there's a plus button on the top right corner, but it just creates a new issue on a selected repository. Is there any chance this button can be for creating a repository or issue in one button via drop-down in the GitHub flavor or similar. 

Galaxy A14 5G Latest update from Google Play checked Dec 3rd 2024 2:00 PM ( Chicago Time US )

**Comment by stevepopovich** [comment]
Hi @PixelatedPurple I have passed this onto the team for consideration. Thanks for writing in!

**Comment by miraclerose311** [comment]
That is serious problem.

**Reply by PixelatedPurple** [reply]
It was the main reason I switched to mainly desktop, using git on the go worked great if I was working on a single project, but when new ideas pop in, gotta go to Google, and open GitHub in a desktop site because the mobile site has a similar issue in accessing the create Repository page. Please note, I'm kinda dumb and I could just be missing it, so, 
Also thx for being devs favorite support team

**Comment by PixelatedPurple** [comment]
Ok so the mobile site has the create Repository option when you click the user profile icon ( top right ) the menu appears and it's somewhat in the middle. 

Just wanted to clarify because I said the mobile site has the same issue when it does not

Did tell you I was kinda dumb
It be what's right in front of you that you miss. 

So thank you very much GitHub team. Y'all 💯

**Comment by HenSquared** [comment]
Hey @PixelatedPurple now you can create repos natively in the mobile app. Hope that helps you!

https://github.blog/changelog/2026-05-11-create-repositories-on-the-go-with-github-mobile/

**Reply by stevepopovich** [reply]
Nice work!!
